examples/conc_col_pmm/tests/visual_tests/visual_test_pmm.py

```diff
@@ -1,10 +1,9 @@
 from efficalc import Input
 
-from ...calc_document.plotting import get_pmm_data, pmm_plotter_plotly
-from ...col import assign_max_min
+from ...calc_document.calculation import calculation
+from ...calc_document.plotting import pmm_plotter_plotly
```

Comment:
These imports need to remain relative in order to allow this package to operate outside of the efficalc repo. The files are still runnable with the instructions I left in the Readme